packages/sdk/src/auth/services/storage-service.ts

```diff
@@ -1,128 +1,74 @@
-  }
-
-  async setItem(key: string, value: string): Promise<void> {
-    this.storage.set(key, value);
-  }
-
-  async getItem(key: string): Promise<string | null> {
-    return this.storage.get(key) || null;
-  }
-
-  async removeItem(key: string): Promise<void> {
-    this.storage.delete(key);
-  }
-
-  async clear(): Promise<void> {
-    this.storage.clear();
-  }
-}
-
-/**
- * Browser-based storage implementation using localStorage
- */
-export class BrowserStorage implements IStorage {
-  async setItem(key: string, value: string): Promise<void> {
-    try {
-      localStorage.setItem(key, value);
-    } catch (error) {
-      console.error('Error storing data in localStorage:', error);
-      throw error;
-    }
-  }
-
-  async getItem(key: string): Promise<string | null> {
-    try {
-      return localStorage.getItem(key);
-    } catch (error) {
-      console.error('Error retrieving data from localStorage:', error);
-      throw error;
-    }
-  }
-
-  async removeItem(key: string): Promise<void> {
-    try {
-      localStorage.removeItem(key);
-    } catch (error) {
-      console.error('Error removing data from localStorage:', error);
-      throw error;
-    }
-  }
-
-  async clear(): Promise<void> {
-    try {
-      localStorage.clear();
-    } catch (error) {
-      console.error('Error clearing localStorage:', error);
-      throw error;
-    }
-  }
-}
-
-/**
- * Main storage class that handles JWT storage across different platforms
- */
 export class Storage {
   private static readonly JWT_KEY = 'vincent_jwt';
-  private storage: IStorage;
+  private isBrowser: boolean;
 
-  constructor(storageImplementation?: IStorage) {
-    // If no storage implementation is provided, try to detect the environment
-    if (!storageImplementation) {
-      if (typeof window !== 'undefined' && window.localStorage) {
-        this.storage = new BrowserStorage();
-      } else {
-        this.storage = new MemoryStorage();
-      }
-    } else {
-      this.storage = storageImplementation;
-    }
+  constructor() {
+    this.isBrowser = typeof window !== 'undefined' && window.localStorage !== undefined;
   }
 
   /**
    * Store a JWT token
    * @param jwt - The JWT string to store
    */
   async storeJWT(jwt: string): Promise<void> {
-    await this.storage.setItem(Storage.JWT_KEY, jwt);
+    if (!this.isBrowser) {
+      console.warn('Storage operations are not supported in Node environment');
+      return;
+    }
+    try {
+      localStorage.setItem(Storage.JWT_KEY, jwt);
+    } catch (error) {
+      console.error('Error storing JWT in localStorage:', error);
+      throw error;
+    }
   }
 
   /**
    * Retrieve the stored JWT token
    * @returns The stored JWT string or null if not found
    */
   async getJWT(): Promise<string | null> {
-    return await this.storage.getItem(Storage.JWT_KEY);
+    if (!this.isBrowser) {
+      console.warn('Storage operations are not supported in Node environment');
+      return null;
```

Comment:
This should not just return null. The entire implementation is broken in this case; the error should bubble up to the user.

Rather than repeating console.warn statements in the functions defined here that may not be seen, it would be preferable to throw an error from the class constructor so that the developer is immediately aware that they are trying to run code in an unsupported environment.